cs/dnc/src/json/project.json

```diff
@@ -37,20 +37,28 @@
     "dependencies": {
         "attributes": "1.0.0-*",
         "core": "1.0.0-*",
-        "Newtonsoft.Json": {
-            "version": "9.0.1"
-        },
         "reflection": "1.0.0-*"
     },
     "frameworks": {
         "netstandard1.0": {
             "dependencies": {
-                "NETStandard.Library": "1.6.0"
+                "NETStandard.Library": "1.6.0",
+                "Newtonsoft.Json": {
+                    "version": "9.0.1"
+                }
             }
         },
         "netstandard1.6": {
+            "buildOptions": {
+                "define": [
+                    "SUPPORTS_BIGINTEGER"
+                ]
+            },
             "dependencies": {
-                "NETStandard.Library": "1.6.0"
+                "NETStandard.Library": "1.6.0",
+                "Newtonsoft.Json": {
+                    "version": "10.0.1"
```

Comment:
I'm a little confused. Version 10.0.1 doesn't appear anywhere else in this file - looks like the only thing referenced is v9.0.1, regardless of portable/non. What am I missing?